src/classes/CON_Phone.cls

```diff
@@ -324,14 +324,17 @@ public with sharing class CON_Phone {
             String prefix = (String.isNotBlank(namespace)) ? namespace + '__' : '';
             List<Schema.PicklistEntry> preferredPhonePicklist = UTIL_Describe.getFieldPicklistEntries('Contact', prefix+'PreferredPhone__c');
             List<String> preferredPhoneValues = new List<String>();
+            Map<String, String> preferredLabelToApi = new Map<String, String>();  
             for (Schema.PicklistEntry eachValue: preferredPhonePicklist) {
-                preferredPhoneValues.add(eachValue.getValue());
+                PreferredLabelToApi.put(eachValue.getLabel(),eachValue.getValue());
             }
```

Comment:
@skosgi1 ~ Just to confirm, will `eachValue.getValue()` get the picklist's element API value?

Reply:
That's the way I read this as well.

Reply:
correct! getValue() is the API and getLabel() is the field label.